osmo_jupyter/spectrometer.py

```diff
@@ -0,0 +1,107 @@
+import pandas as pd
+import io
+
+
+def _import_spectrometer_txt(spectrometer_data_file_path):
+    with open(spectrometer_data_file_path, 'r', newline=None) as infile:
+        for line in infile:
+            if line.strip() == '>>>>>Begin Spectral Data<<<<<':
+                file_content_with_unix_endings = infile.read()
+                unformatted_spectrometer_data = pd.read_csv(
+                    io.StringIO(file_content_with_unix_endings),
+                    sep='\t',
+                    lineterminator='\n',
+                )
+
+                return unformatted_spectrometer_data
+        raise ValueError('Spectrometer data not in expected format; check the file header')
+
+
+def _clean_up_spectrometer_data_header(unformatted_spectrometer_data):
+
+    ''' Cleans up header of raw spectrometer data, and removes redundant timestamp column.
+
+    Args:
+        unformatted_spectrometer_data: a pandas df of original spectrometer data,
+            imported with _import_spectrometer_txt()
+
+    Return:
+        A pandas dataframe with a sensible header, and only one timestamp column
+    '''
+
+    timestamp_column, epoch_column = unformatted_spectrometer_data.columns[[0, 1]]
+    spectrometer_data_with_clean_header = unformatted_spectrometer_data.rename(
+        # timestamp column is in local time, epoch_time column is in UTC
+        columns={
+            timestamp_column: 'timestamp',
+            epoch_column: 'epoch_time'
+        }
+    )
+    # Remove redundant timestamp column
```

Comment:
nit: timestamp --> epoch_time

Reply:
if you do this, it'd be nice to explicitly point out that it's redundant to the timestamp column
```suggestion
    # 'epoch_time' column is redundant to 'timestamp' column. Remove it
```